.github/scripts/release/.release-it.json

```diff
@@ -0,0 +1,34 @@
+{
+  "git": {
+    "commitMessage": "chore: release v${version}",
+    "tagName": "${version}",
+    "changelog": "npx auto-changelog --stdout --commit-limit false --unreleased --template https://raw.githubusercontent.com/release-it/release-it/master/templates/changelog-compact.hbs"
+  },
+  "github": {
+    "release": true,
+    "releaseName": "${version}"
+  },
+  "npm": {
+    "publish": false,
+    "ignoreVersion": true
+  },
+  "plugins": {
+    "@release-it/bumper": {
+      "in": {
+        "file": "../../../VERSION",
+        "type": "text/plain"
+      },
+      "out": {
+        "file": "../../../VERSION",
+        "type": "text/plain"
+      }
+    }
+  },
+  "hooks": {
+    "after:bump": [
+      "npx auto-changelog -p -o ../../../CHANGELOG.md --hide-empty-releases",
```

Comment:
More of a question, but how come changelog is defined twice (in the git section and here in after:bump). Can we remove the one from the git section above? I was curious about this when I was doing the dart one too and wasn't sure.